tests/test_databases.py

```diff
@@ -303,24 +303,26 @@ async def test_ddl_queries(database_url):
             await database.execute(query)
 
 
+class DBException(Exception):
+    pass
```

Comment:
Maybe we don't need this?

Reply:
I just kept what we had before, you're right we could remove that!

Reply:
I've applied your patch & removed the redundant class

Reply:
Great, thanks.